src/components/shared/MotionFadeDown.tsx

```diff
@@ -0,0 +1,19 @@
+import { motion } from 'framer-motion';
+import type { ReactNode } from 'react';
+
+type Props = {
```

Comment:
[K3]: 타입명 수정 부탁드립니다.
```suggestion
type MotionFadeDownProps = {
```